client/src/lib/api.js

```diff
@@ -68,6 +69,40 @@ api.collectProductInfo = (id) => {
   .catch(err => console.log(err));
 }
 
+// QUESTIONS AND ANSWERS
+api.listQuestions = (params) => {
+  return get('/qa/questions', params);
+}
+
+api.listAnswers = (question_id) => {
+  return get(`/qa/questions/${question_id}/answers`, params);
+}
+
```

Comment:
line 78 I would remove the params argument since the function isn't accepting any params as a parameter.